pkgs/misc/vim-plugins/overrides.nix

```
@@ -515,6 +516,11 @@ self: super: {
     configurePhase = "cd vim";
   });
 
+  parinfer-rust = buildVimPluginFrom2Nix {
+    src = parinfer-rust;
+    inherit (parinfer-rust) version pname;
+  };
```

Package already exports it, and the runtime is smaller
```suggestion
  parinfer-rust = pkgs.parinfer-rust;
```